alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -107,6 +107,7 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
 
         val updatedLastRunContext = lastRunContext.toMutableMap()
 
+        val queryIdToDocIds = mutableMapOf<String, MutableSet<String>>()
```

Comment:
This is the same as `inputRunResults`, cant we use this instead of recreating it?